src/docfx/Errors.cs

```diff
@@ -840,5 +840,18 @@ public static class SourceMap
             public static Error DuplicateSourceMapItem(string key, IEnumerable<PathString> originalFiles)
                 => new Error(ErrorLevel.Warning, "duplicate-source-map-item", $"'{key}' is duplicated from {StringUtility.Join(originalFiles)}");
         }
+
+        public static class ApplyTemplate
+        {
+            /// <summary>
+            /// Template path is not specified.
+            /// </summary>
+            /// Behavior: ❌ Message: ❌
+            public static Error TemplateNotSpecified()
```

Comment:
This isn't also a user error. If the command line argument is required, set `System.CommandLine.Option.IsRequired = true`.